### ISSUE TITLE
Localizing Strings in iOS Info.Plist in MAUI Single Project Not Working

### ISSUE THREAD

**Issue by TonyLugg** [xamarin/xamarin-macios#18429]
### Steps to Reproduce

1.  Add iOS lproj folders to iOS platform for InfoPlist.strings localization.
2.  Run app on devices using English, Spanish, French
3.  Localization does not work

*** NOTE:  This worked in Xamarin iOS project of Xamarin Forms solution.

### Expected Behavior

When first running the app, attempting to access location services prompts the user for permission using strings in info.plist.  These strings should be localized based on the current language of the device.

### Actual Behavior

Always showing English (base language)

### Environment

<details>
<summary>.NET 7 MAUI</summary>

```
Microsoft Visual Studio Professional 2022
Version 17.6.2
VisualStudio.17.Release/17.6.2+33723.286
Microsoft .NET Framework
Version 4.8.09032

Installed Version: Professional

Visual C++ 2022   00476-80000-00000-AA557
Microsoft Visual C++ 2022

ADL Tools Service Provider   1.0
This package contains services used by Data Lake tools

ASA Service Provider   1.0

ASP.NET and Web Tools   17.6.326.62524
ASP.NET and Web Tools

Azure App Service Tools v3.0.0   17.6.326.62524
Azure App Service Tools v3.0.0

Azure Data Lake Tools for Visual Studio   2.6.5000.0
Microsoft Azure Data Lake Tools for Visual Studio

Azure Functions and Web Jobs Tools   17.6.326.62524
Azure Functions and Web Jobs Tools

Azure Stream Analytics Tools for Visual Studio   2.6.5000.0
Microsoft Azure Stream Analytics Tools for Visual Studio

C# Tools   4.6.0-3.23259.8+c3cc1d0ceeab1a65da0217e403851a1e8a30086a
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Extensibility Message Bus   1.4.21 (main@8f226a8)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

Microsoft Azure Hive Query Language Service   2.6.5000.0
Language service for Hive query

Microsoft Azure Stream Analytics Language Service   2.6.5000.0
Language service for Azure Stream Analytics

Microsoft Azure Tools for Visual Studio   2.9
Support for Azure Cloud Services projects

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Mono Debugging for Visual Studio   17.6.41 (790a401)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   6.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

Office Developer Tools for Visual Studio   17.6.33520.00
Microsoft Office Developer Tools for Visual Studio

Razor (ASP.NET Core)   17.6.0.2326105+37f2727000f6a964584cafb86e69cf0b39baecb7
Provides languages services for ASP.NET Core Razor.

SQL Server Data Tools   17.6.13.0
Microsoft SQL Server Data Tools

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TypeScript Tools   17.0.20329.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.6.0-3.23259.8+c3cc1d0ceeab1a65da0217e403851a1e8a30086a
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   17.6.0-beta.23174.5+0207bea1afae48d9351ac26fb51afc8260de0a97
Microsoft Visual F# Tools

Visual Studio IntelliCode   2.2
AI-assisted development for Visual Studio.

VisualStudio.DeviceLog   1.0
Information about my package

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

VSPackage Extension   1.0
VSPackage Visual Studio Extension Detailed Info

Workflow Manager Tools 1.0   1.0
This package contains the necessary Visual Studio integration components for Workflow Manager.

Xamarin   17.6.0.251 (d17-6@318364c)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   17.6.6.0 (remotes/origin/d17-6@cb430751d1)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   17.6.32 (98c12a2)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   13.2.0.6 (d17-5/a200af1)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: 6dd9def
    Java.Interop: xamarin/java.interop/d17-5@149d70fe
    SQLite: xamarin/sqlite/3.40.1@68c69d8
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d17-5@9f02d77


Xamarin.iOS and Xamarin.Mac SDK   16.4.0.6 (97731c92c)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

</details>

### Example Project (If Possible)

https://github.com/TonyLugg/MauiTrackTestSP-With-info.plist-localization

**Comment by rolfbjarne** [comment]
I can reproduce this, and I was able to work around it by adding the following to the project file:

```xml
<PropertyGroup>
    <IPhoneResourcePrefix>Platforms/iOS/Resources</IPhoneResourcePrefix>
</PropertyGroup>
```

MAUI already sets this property, but it's doing so too late in the build property. However, this can probably be fixed on our side, without any changes to MAUI - we'd just have to make sure to read the property later.

**Comment by TonyLugg** [comment]
I had tried that IPhoneResourcePrefix setting before and it did not work.  I tried it again and it is still not working for me.  When the location tracking permission string is displayed, it is still in English.

**Comment by TonyLugg** [comment]
@rolfbjarne I really need to get this worked out, it's stopping deployment to Apple.  Can you provide the repo of the solution you got working?

**Comment by rolfbjarne** [comment]
> Can you provide the repo of the solution you got working?

I used your repo, with this change:

```xml
$ git diff
diff --git a/MauiTrackTestSP.csproj b/MauiTrackTestSP.csproj
index 9083c42..087b175 100644
--- a/MauiTrackTestSP.csproj
+++ b/MauiTrackTestSP.csproj
@@ -27,6 +27,8 @@
                <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">10.0.17763.0</SupportedOSPlatformVersion>
                <TargetPlatformMinVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">10.0.17763.0</TargetPlatformMinVersion>
                <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">6.5</SupportedOSPlatformVersion>
+
+               <IPhoneResourcePrefix>Platforms/iOS/Resources</IPhoneResourcePrefix>
        </PropertyGroup>

        <PropertyGroup Condition="'$(TargetFramework)'=='net7.0-ios'">
```

Can you try building on a Mac to see if that makes a difference (and if that would unblock you)?

**Comment by TonyLugg** [comment]
@rolfbjarne I copied the solution to the Mac and updated VS Mac but the solution fails to load.  No explanation, just says Load Operation Failed.  Do I need to disable Windows?  Sorry, not a Mac guy, never had to do this before.  Is there a command line build option?

**Comment by rolfbjarne** [comment]
@TonyLugg just to confirm, I believe you found a workaround?

Re: https://discord.com/channels/732297728826277939/732297808148824115/1118630647137837156

**Comment by TonyLugg** [comment]
Yes, the workaround you provided is doing the trick.  I moved the lproj folders from Platforms\iOS\Resources to \Resources and made the ItemGroup conditional to iOS:

```
	<ItemGroup Condition="'$(TargetFramework)'=='net7.0-ios'">
	  <BundleResource Include="Resources\Base.lproj\Localizable.strings" />
	  <BundleResource Include="Resources\es.lproj\InfoPlist.strings" />
	  <BundleResource Include="Resources\es.lproj\Localizable.strings" />
	  <BundleResource Include="Resources\fr.lproj\InfoPlist.strings" />
	  <BundleResource Include="Resources\fr.lproj\Localizable.strings" />
	</ItemGroup>
```

It would be better if the MAUI build process would pickup the iOS folders from Platforms\iOS\Resources like we had with our separate iOS project (Xamarin Forms), so hopefully this will be fixed in the future.

**Comment by rolfbjarne** [comment]
> It would be better if the MAUI build process would pickup the iOS folders from Platforms\iOS\Resources like we had with our separate iOS project (Xamarin Forms), so hopefully this will be fixed in the future.

Yes, I don't think this will be particularly hard to fix, so it should be fixed in our .NET 8 release.

**Comment by rolfbjarne** [comment]
This turned out to be somewhat trickier than I expected, so it seems it'll be easier to fix in MAUI. As such, I'm closing this as a duplicate of https://github.com/dotnet/maui/issues/16734.
